specification/msi/resource-manager/Microsoft.ManagedIdentity/2015-08-31-preview/ManagedIdentity.json

```diff
@@ -0,0 +1,535 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2015-08-31-preview",
+    "title": "ManagedServiceIdentityClient",
+    "description": "The Managed Service Identity Client."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.ManagedIdentity/operations": {
+      "get": {
+        "operationId": "Operations_List",
```

Comment:
Operations that return arrays should be marked as [pageable](https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-pageable)

Reply:
Does that mean we also have to provide a nextLink? Our list of operations is very small and might not require paging

Reply:
If paging might happen, yes, a nextLink is neded.
Even if the operation is a single page, it still benefits from marking it as paged.
Look here for example 3
https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-pageable

Reply:
Is it ok to mark the operation as pageable in Swagger and then not return a nextLink? I guess not.
If we mark it as pageable then we would at least have to return a null nextLink (if the result is single page) Is that right?

Reply:
For paged results, when the response can be contained in a page, should the nextLink not be returned at all or nextLink: null should be returned?

Reply:
Marked it pageable